apps/timetable/admin/ui/css/skin.css

```diff
@@ -49,6 +49,15 @@
 }
 
 
+/***********/
+/* POPOVER */
+/***********/
+
+.popover {
```

Comment:
Same here, check whether or not this interferes with the Bootstrap popover class